index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400&display=swap" rel="stylesheet">
+
+
+    <link rel="stylesheet" href="style.css">
+
+
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <div class="status-bar">
+            <div class="time-section">
+                <a href="#">
+                    <h2>9:41</h2>
+                </a>
+            </div>
+            <div class="right-section">
+                <img src="img/CellularConnection.png" alt="cellularconnection">
+                <img src="img/Wifi.png" alt="wifi">
+                <img src="img/Battery.png" alt="battery">
+            </div>
+        </div>
+        <div class="toolbar">
+            <div class="logo-section">
+                <h2>My Logo</h2>
+            </div>
+            <div class="icon-menu">
+                <img src="img/Union.png" alt="union">
+            </div>
+        </div>
+    </header>
+    <div class="body-section">
```

Comment:
# [OPTIONAL]
- [ ] Please consider changing this `div` tag to a `section` tag, this will make your project more semantic.
![image](https://user-images.githubusercontent.com/73256642/133630590-93838b7e-21d3-451f-bca6-9fc65d05cb3e.png)